srcpkgs/remake/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'remake'
+pkgname=remake
+version=4.3
+revision=1
+build_style=gnu-configure
+makedepends="gettext-devel readline-devel make"
```

Comment:
Why is `make` here? It's installed in the chroot by default.

Reply:
I didn't know that it's installed (this is the first time I am trying to add a package to void-packages). I have updated the template and removed `make`. By the way, how can I inspect what all packages are already installed in chroot by default?

Reply:
`./xbps-src clean && ./xbps-src chroot`, then `xbps-query -l`